app/views/course/assessment/question/voice_responses/new.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+json.partial! 'form', locals: {
+  question: @voice_response_question,
```

Comment:
Do we need to pass the variables from line 3 to 5?

Reply:
after I understand what it actually entails, I concluded it's not needed. I have removed it in the commit that will follow